frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -125,14 +126,29 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
     propagationPolicy: 'Foreground',
   };
 
-  const uninstallOperator = React.useCallback(() => {
+  const uninstallOperator = React.useCallback(async () => {
     const patch = removePlugins
       ? getPatchForRemovingPlugins(consoleOperatorConfig, enabledPlugins)
       : null;
 
+    const clusterServiceVersionExists = async () => {
+      try {
+        await k8sGetResource({
+          model: ClusterServiceVersionModel,
+          name: subscription.status.installedCSV,
+          ns: subscription.metadata.namespace,
+        });
+        return true;
+      } catch (err) {
+        // eslint-disable-next-line no-console
+        console.log(err.message);
+        return false;
+      }
+    };
```

Comment:
What happens when the request fails with something other than a 404?

Reply:
Good point.  We need to report out those other errors.

/hold